### PR TITLE
Pac-Man: Polishing round 1

### PR DESCRIPTION
**Pull request by frrrances**
Small polishing on the kitchen sink and themes.

### CODE REVIEW

File: src/themes/brutalist.css [hunk 1]
```diff
@@ -351,6 +351,35 @@
...
   }
 
+  pre,
+  code {
+    font-size: var(--wa-font-size-m);
+    background-color: var(--wa-color-neutral-fill-subtle);
+  }
+
+  /* syntax highlighting */
+  .token {
```
Comment (frrrances) [inline src/themes/brutalist.css:361]: The default syntax highlighting colors were close but looked a bit duller, so I added these overrides. If we don't want overrides, we can probably just do without them and use the default styling. (Same for all the syntax highlighting overrides below...)

File: docs/pages/experimental/kitchen-sink.md [hunk 2]
```diff
@@ -1168,13 +1169,23 @@ toc: false
...
+  }
+
   wa-input::part(input) {
     width: 100%;
   }
 
   .square-frame,
   .landscape-frame {
     overflow: hidden;
+    border-radius: calc(var(--wa-corners-m) - var(--wa-panel-border-width));
```
Comment (lindsaym-fa) [inline docs/pages/experimental/kitchen-sink.md]: We have some rounding sneaking into the `wa-card` images from this style.

Screenshot 2024-02-15 at 6 16 36 PM
Screenshot 2024-02-15 at 6 16 57 PM
Screenshot 2024-02-15 at 6 19 32 PM

**Review by lindsaym-fa — CHANGES_REQUESTED** [review]
Changes look great! Just one small thing to touch up.

**Review by lindsaym-fa — APPROVED** [review]
All looks good to me! 🤩

**Pull request merged**